.depcheckrc

```diff
@@ -0,0 +1,5 @@
+ignores: [
+  "@types/*",    # Ignore TypeScript type definitions
```

Comment:
`depcheck` complains about `@types/pg`. If I remove that, everything still seems to be fine. It does _not_ complain about `@types/mocha`, so it's not that it in general doesn't understand types. I think we can remove this line as well, and thereby the whole file.